bidwire/notifier.py

```diff
@@ -4,7 +4,7 @@
 from bid import Bid, get_bids_from_last_n_hours
 from db import Session
 from document import Document, get_docs_from_last_n_hours
-from notifiers.cityofboston_notifier import CityOfBostonNotifier
```

Comment:
Please revert the file rename of `cityofboston_notifer` and all related diffs, since they are not pertinent to your new site scraper+notifier.